graph-kiota/graph-kiota.md

```diff
@@ -0,0 +1,126 @@
+### Graph OpenAPI specification
```

Comment:
```suggestion
### Graph OpenAPI description
```
The specification is the document that describes OpenAPI syntax.  API descriptions are what are created to describe APIs using the OpenAPI specification.